File: services/treadmillState.py
class TreadmillState:

    # The treadmill is not running
    Ready = 1

    # The 3, 2, 1 start countdown is in progress
    Starting = 2

    # The treadmill is running
    Started = 3

    # The treadmill is paused (could still be rolling)
    Paused = 4

    # Workout has ended, summary is shown
    Summary = 5

File: Core/Treadmill.Models/TreadmillState.cs
namespace Treadmill.Models
{
  public enum TreadmillState
  {
    /// <summary>
    /// The treadmill is not running
    /// </summary>
    Ready = 1,

    /// <summary>
    /// The 3, 2, 1 start countdown is in progress
    /// </summary>
    Starting = 2,

    /// <summary>
    /// The treadmill is running
    /// </summary>
    Started = 3,

    /// <summary>
    /// The treadmill is paused (could still be rolling)
    /// </summary>
    Paused = 4,

    /// <summary>
    /// Workout has ended, summary is shown
    /// </summary>
    Summary = 5,
  }
}
